lib/Sema/CSSimplify.cpp

```diff
@@ -298,6 +299,47 @@ matchCallArguments(SmallVectorImpl<AnyFunctionType::Param> &args,
     if (numClaimedArgs == numArgs)
       return None;
 
+    /// When we determine which argument is bound to unlabeled parameter,
+    /// consider still unbounded parameter which is prior to current parameter.
+    /// In order not to intersect binding position that remaining parameter will
+    /// bind later, skip a unlabeled argument as much as it can.
+    ///
+    /// For example:
+    /// @code
+    /// func f(aa: Int, _ bb: Int) {}
+    /// f(0, 1)
+    /// @endcode
+    /// Choice argument[1] for parameter[1] so that parameter[0] will be bounded
+    /// to argument[0] later.
+    ///
+    /// Because variadics parameter can be bounded with more than one arguments,
+    /// they don't this.
```

Comment:
Let's re-phrase - `Avoid considering variadic parameters because such parameters could be bound to one than one argument`.

Reply:
Thanks!